iFixit/react-commerce
# Update footer to the new PageWrapperLayout

This PR moves from the work done by @aburke07 to on the footer to include the new Wrapper component.
Together with @mmarcon91 we also added also some additional styles to avoid situations in which portions of the ui ended up being to close to each other.

### QA

1. Visit [Vercel preview](https://react-commerce-git-fix-footer-details-ifixit.vercel.app/Parts)
2. Check that - tanks to the new wrapper - the footer is now wide as much as the main content of the page
